src/lib/utils.js

```diff
@@ -181,6 +181,7 @@ function getRefinements(results, state) {
   forEach(state.numericRefinements, (operators, attributeName) => {
     forEach(operators, (values, operator) => {
       forEach(values, (name) => {
+        name = name + '';
```

Comment:
I do not know either, I thought you coded the getRefinements util for the current refined values widget that's why I though you knew more maybe?

Since it's currently only exposed through the "name" attribute in the current refined value template, I would go for a string since it is stringified by default.

You could pass a transformData and expect name to be a number but I think most of the cases this is not what you want. Even if people want the original value we can add it later on in a "value" prop